src/server/index.js

```diff
@@ -1,3 +1,8 @@
+// dotenv is used only in development & test
```

Comment:
why is this addition (in this file) needed at all?

It would be better to keep dotenv stuff out of src/ and more in build/startup contexts.

Reply:
In this case, it's a runtime issue. The config needs to be loaded here because it enables debug mode in this file.

Reply:
It shouldn't.  I know it's a little labyrinthine, but if you run `yarn dev` that runs (see package.json) dev-tools/Procfile.dev which runs `./dev-tools/babel-run ./webpack/server.js` and  webpack/server.js loads webpack/config.js which *does* load dotenv.

The current setup actually will never load dotenv to begin with (and that's intentional) in production, because production runs server/index.js directly (or a built version of it).

So actually, is this PR required at all?

Reply:
Harpo had this as his fix a long time ago to address #753 we still see that problem when trying to set up people now. Are you sure that that full flow is actually executed when running `yarn dev`?

Reply:
If it's not, then we should solve the core problem rather than adding additional code that make it even more confusing.